### DISCUSSION TITLE
How to see the number of commits before and behind in the fork list view.

### DISCUSSION THREAD

**Discussion by ArminJo** [community/community#22601] [New to GitHub]
<p>Hello,</p>
<p>I don’t know how to request a feature to be added to GitHub so thought I’d post it here:</p>
<p>I often look for projects wich are poorly mantained, but have a lot of forks, like <a href="https://github.com/going-digital/Talkie" rel="nofollow noopener">https://github.com/going-digital/Talkie</a></p>
<p><img src="https://user-images.githubusercontent.com/3881884/181072136-d2aead52-7932-4870-a05e-c8351b2ac7f0.png" alt="Unbenannt.PNG" data-base62-sha1="tNGnTVEZLUpGaqMBr146iMOJQA8" width="238" height="235"></p>
<p>To decide which of the forks is the best candidate for containing the missing features/bugfixes it would be of great value to see the numbers of commits before and behind of each fork directly in the fork list at each element.</p>
<p>Until now I must browse all forks to get this info :-(.</p>
<p>Below is a proposal how I would like to see the forks list.</p>
<p><img src="https://user-images.githubusercontent.com/3881884/181072137-96fbb6d5-541b-4bba-bf79-3286c86ac1f2.png" alt="Forks.PNG" data-base62-sha1="vePMm9o3mmui7wIjfoVpvRAu9QY" width="238" height="235"></p>

**Comment by Wabri** [comment]
<p>Maybe you can try to send the request with this form: <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a></p>

**Comment by ArminJo** [comment]
<p>Thanks,</p>
<p>I sent the request now. Maybe it helps.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi ArminJo,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>
<p>Andrea</p>

**Comment by ArminJo** [comment]
<p>Hello Andrea,</p>
<p>thanks a lot to put the request on yout list. Maybe I will see it next year for christmas :slight_smile:</p>
<p>But to be honest, there must be any solution to cope with all the old stuff Github aquired over time.</p>
<p>Have a nice Weekend</p>
<p>Armin Joachimsmeyer</p>
